### ISSUE TITLE
[Review] Using wastewater data to monitor SARS-CoV-2 in England

### ISSUE THREAD

**Issue by mhauru** [alan-turing-institute/TuringDataStories#207]
# Story Review: Using wastewater data to monitor SARS-CoV-2 in England

**Story Name:** Using wastewater data to monitor SARS-CoV-2 in England

**Submitting Author:** @AoifeHughes (Aoife Hughes)

**Pull Request:** https://github.com/alan-turing-institute/TuringDataStories/pull/206

**Reviewers:** @radka-j (Radka Jersakova), @callummole (Callum Mole)

## Reviewer instructions & questions

Radka & Callum, please carry out your review in this issue by updating the checklist below, and writing new comments in case you have any questions. If you cannot edit the checklist please:

- Make sure you're logged in to your GitHub account
- Be sure to accept the invite at this URL: https://github.com/alan-turing-institute/TuringDataStories/settings/access

Any questions, concerns or suggestions regarding the review process please let @crangelsmith, @DavidBeavan or @samvanstroud know.

✨ Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest ✨

#### Code of conduct
- [ ] I confirm that I read and will adhere to the Turing Data Stories [code of conduct](https://github.com/alan-turing-institute/TuringDataStories/blob/master/CODE_OF_CONDUCT.md).

#### General checks
- [ ]  **Notebook:** Is the source code for this data story available as a notebook in the linked pull request?
- [ ]  **Contribution and authorship:** Are the authors clearly listed? Does the author list seem appropriate and complete?
- [ ] **Scope and eligibility:** Does the submission contain an original and complete analysis of open data? Is the story aligned with the Turing Data Stories [vision statement](https://github.com/alan-turing-institute/TuringDataStories#our-vision)?

#### Reproducibility
- [ ] Does the notebook run in a local environment?
- [ ] Does the notebook build and run in binder?
- [ ] Are all data sources openly accessible and properly cited with a link?
- [ ] Are the data [open](https://opendatahandbook.org/guide/en/what-is-open-data/), and do they have an explicit licence, provenance and attribution?

#### Pedagogy
- [ ] Does the story demonstrate some specific data analysis or visualisation techniques?
- [ ] Are these techniques well motivated?
- [ ] Are these techniques well implemented?
- [ ] Is the notebook well documented, using both markdown cells and comments in code cells?
- [ ] Does the notebook have an introduction section motivating the story?
- [ ] Does the notebook have a conclusion section discussing the main insight from the stories?
- [ ] Is the paper well written (it does not require editing for structure, language, or writing quality)?

#### Context
- [ ] Does the story give an insight into some societal issue?
- [ ] Is the context around this issue well referenced (newspaper articles, scientific papers, etc.)?

#### Ethical 

- [ ] Is any linkage of datasets in the story unlikely to lead to an increased risk of the personal identification of individuals?
- [ ] Is the Story truthful and clear about any limitations of the analysis (and potential biases in data)?
- [ ] Is the Story unlikely to lead to negative social outcomes, such as (but not limited to) increasing discrimination or injustice?

## AOB

If you have comments pertaining to particular parts of the story, please use the ReviewNB app to submit them.

Feel free to agree between yourselves on how to divide the review work.

**Comment by mhauru** [comment]
https://github.com/all-contributors please add @radka-j  for review

**Comment by mhauru** [comment]
@all-contributors please add @callummole for review

**Comment by mhauru** [comment]
@all-contributors please add @radka-j for review

**Comment by mhauru** [comment]
@all-contributors please add @radka-j for review

**Comment by mhauru** [comment]
@all-contributors please add @AoifeHughes for ideas

**Comment by mhauru** [comment]
@all-contributors please add @AoifeHughes for content

**Comment by mhauru** [comment]
@all-contributors please add @AoifeHughes for code

**Comment by mhauru** [comment]
@all-contributors please add @AoifeHughes for data

**Comment by mhauru** [comment]
No, you can tick boxes in that main checklist in the issue body. This way if you've checked some aspect and ticked it, Radka will know she doesn't have to.

**Comment by crangelsmith** [comment]
> No, you can tick boxes in that main checklist in the issue body. This way if you've checked some aspect and ticked it, Radka will know she doesn't have to.

Just to add to this, it makes sense on the reproducibility side that only one reviewer carries out the checks, but I think in the other sections both reviewers must agree that the requirement is fulfilled, given that it can be a bit more subjective.

**Comment by callummole** [comment]
I've reviewed the notebook and posted a bunch of comments. Well done, @AoifeHughes, it's looking really nice. Most of my comments are about the modelling.

**Comment by AoifeHughes** [comment]
Thanks for doing so quickly and throughly @callummole, I’ll go through and address soon! 😊

**Comment by AoifeHughes** [comment]
Going to address things here and will update this list as I push updates, will notify when complete

For @AoifeHughes todo:
- [ ] Address grammar 
  - [ ] https://github.com/alan-turing-institute/TuringDataStories/pull/206/commits/e01e9de42b98f220eac53f233992b071d327ae68 does first paragraph and fixes filenames
- [ ] Add links to additional instructional files
- [ ] Clarity and conciseness
